docs/proposals/validation-tools/readme.md

```diff
@@ -0,0 +1,83 @@
+# Swagger Validation Tools VS Code Extension
+
+## Objectives and High-level plan
+
+We plan on building a one-stop shop vs-code extension that will help the swagger authors get live and prompt feedback on the quality and correctness of their swaggers. We plan on reusing and building on top of the work that was done for the Autorest Linter VS code extension and integration our tools Autorest to simplify the users experience.
+
+## Overall architecture
+
+The following diagram provides a high-level architecture of the proposed SVT (Swagger Validation tools) VS Code extension.
+
+> ![](./diagram.png)
+
+1. SVT VS Code extension&#39;s client: the client is the implementation of UI and experience of the swagger authoring, details will follow later in the document on the proposed user experience. The swagger tooling team will take ownership of this component.
+
+2. SVT VS Code server component: The Autorest team will be owning this component and the current plan is to:
+

```

Comment:
We may want to add some of the considerations we mentioned offline, like the ability for AutoRest to run on any platform, the fact that the current extension is triggered by 'any' json file load (not just swagger file).